lib/db/mysql.js

```diff
@@ -199,14 +199,16 @@ module.exports = function (log, error) {
         sessionToken.uaBrowserVersion,
         sessionToken.uaOS,
         sessionToken.uaOSVersion,
-        sessionToken.uaDeviceType
+        sessionToken.uaDeviceType,
+        sessionToken.tokenVerificationId
```

Comment:
The transaction fails because `tokenVerificationId` is `BINARY(32) NOT NULL`, but on adding tests I discovered that the memory db wasn't failing the same way (I expected it to because it dereferenced `null`). So your reputation for sniffing out hidden errors continues unabated.

Tests and fix added in https://github.com/mozilla/fxa-auth-db-mysql/pull/132/commits/b6227155d1bcbae9f6a1d9b19467de3788385163.